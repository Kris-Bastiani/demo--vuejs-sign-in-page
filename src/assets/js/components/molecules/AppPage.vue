<template lang="pug">
	div
		AppHeader
		slot
</template>

<script>
import AppHeader from './AppHeader.vue';

export default {
	components: { AppHeader },
}
</script>
