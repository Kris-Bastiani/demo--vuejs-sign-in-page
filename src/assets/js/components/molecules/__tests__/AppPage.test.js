import { shallowMount } from '@vue/test-utils';
import AppPage from '../AppPage.vue';

describe('AppPage', () => {
	it('Matches snapshot', () => {
		const wrapper = shallowMount(AppPage);
		expect(wrapper.element).toMatchSnapshot();
	});
});
