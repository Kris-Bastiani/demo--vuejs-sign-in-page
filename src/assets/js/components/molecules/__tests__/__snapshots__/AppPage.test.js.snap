// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppPage Matches snapshot 1`] = `
<div>
  <appheader-stub />
</div>
`;
